src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Bad inheritance: Lottery is using ColorSupplier to get a random ball color, but you don't need inheritance for It. You can just call the method, as this method is static you can call it without creating a ColorSupplier object.
Link to read about static methods: https://www.baeldung.com/java-static-methods-use-cases
Link to read about inheritance: https://www.w3schools.com/java/java_inheritance.asp